source/speech/__init__.py

```diff
@@ -766,8 +769,25 @@ def _speakTextInfo_addMath(speechSequence, info, field):
 	except (NotImplementedError, LookupError):
 		return
 
-def speakTextInfo(info, useCache=True, formatConfig=None, unit=None, reason=controlTypes.REASON_QUERY, _prefixSpeechCommand=None, onlyInitialFields=False, suppressBlanks=False,priority=None):
+re_white_space = re.compile(r"(\s+|$)", re.DOTALL)
+
+class _TextCommand(str):
+	"""str subclass to distinguish normal text from field text when processing text info speech."""
+
+def speakTextInfo(

```

Comment:
Should we add typing info to these since they're public API methods?

Reply:
I think that can be done, yes.